recipe/meta.yaml

```diff
@@ -1,4 +1,5 @@
 {% set version = "4.5.4" %}
+{% set version_win = "a0e1db7f2de7bdc456648d8d0816f5273afaf3e7" %}
```

Comment:
This doesn't seem to be a great version string. As it is a hash, we will get ill-defined behavior comparing it.

Reply:
Not relevant.

Reply:
Why not? How are they versioning the Windows portion? It doesn't seem to match the Unix portion?

Reply:
This has nothing to do with the suitesparse version. It doesn't appear in the packages or in any of the files. It's only used to select the GitHub commit in the repository. The suitesparse version is still 4.5.4.

Reply:
How do you know?

Reply:
Check the repository. It says so.

Reply:
I did. It was not obvious to me. There were a bunch of releases with different versions all of which mentioned something else other than 4.5.4 compatibility ( [4.5.1 for example]( https://github.com/jlblancoc/suitesparse-metis-for-windows/releases/tag/v1.3.1 ) ). Please provide a link to where you see this info.

Reply:
https://github.com/jlblancoc/suitesparse-metis-for-windows/blob/master/SuiteSparse/README.txt

Reply:
Wait, you're correct.